### ISSUE TITLE
Search box displays search and close icons on top of each other in IE11 RTL

### ISSUE THREAD

**Issue by joswhite** [MicroFocus/ux-ias#18]
![image](https://user-images.githubusercontent.com/7979277/38517355-f6ebfad8-3bf6-11e8-9008-3f19f7209623.png)

Need to replace `right: initial` with `right: auto`. (IE11 doesn't accept `initial`)

````
 .ias-search {
    >.ias-icon-button {
      right: initial;
    }
  }
````

**Comment by joswhite** [comment]
We also need to do this in _tab.scss (use display: block not display: initial) (line 68)

**Comment by joswhite** [comment]
Both fixed by 135c44cb26d10710e3d8276302edd476eb940106
